gulpfile.babel.ts

```diff
@@ -352,7 +358,8 @@ const sourceMapPlugin = new webpack.SourceMapDevToolPlugin({
 if (args.remote) {
   config = {
     ...remoteCompatConfig,
-    devtool: false,
+    devtool: false, // using manually-configured plugin instead
+    plugins: [sourceMapPlugin],
```

Comment:
This shouldn't affect our sentry source map setup, but roughly an hour after a remote deploy, we'll want to check that source map reporting is working as expected there.

Post MV3 at Streak, our source map setup will be simplified as well.

Reply:
I didn't actually make a real change there, I just reordered some lines and added a comment.

Reply:
My mistake.